cdap-api/src/main/java/co/cask/cdap/api/RuntimeContext.java

```diff
@@ -37,12 +37,17 @@
   Map<String, String> getRuntimeArguments();
 
   /**
-   * @return The application namespace
+   * @return The cluster name.
```

Comment:
Ok. I was adding it as some of the @return statement has it (https://github.com/caskdata/cdap/pull/7315/files#diff-02980aab78a135d0df7c0164de33152bR35)